cli/src/nonce.rs

```diff
@@ -597,7 +597,9 @@ pub fn process_show_nonce_account(
         };
         if let Some(data) = data {
             nonce_account.nonce = Some(data.blockhash().to_string());
-            nonce_account.lamports_per_signature = Some(data.fee_calculator.lamports_per_signature);
+            // TODO - shoudl also nonce_account's lamports_per_signature in this PR? or do it in
+            // separate one.
+            nonce_account.lamports_per_signature = None;
```

Comment:
Can remove optional `lamports_per_signature` from `CliNonceAccount` in separate PR, but should we keep it for now as deprecated if that's break cli client?